ee/clickhouse/queries/experiments/trend_experiment_result.py

```diff
@@ -59,11 +60,21 @@ def __init__(
         breakdown_key = f"$feature/{feature_flag.key}"
         variants = [variant["key"] for variant in feature_flag.variants]
 
+        # our filters assume that the given time ranges are in the project timezone.
+        # while start and end date are in UTC.
+        # so we need to convert them to the project timezone
+        if team.timezone:
+            start_date_in_project_timezone = experiment_start_date.astimezone(pytz.timezone(team.timezone))
+            end_date_in_project_timezone = (
+                experiment_end_date.astimezone(pytz.timezone(team.timezone)) if experiment_end_date else None
```

Comment:
yes-ish, they just say 'now' and we convert it into UTC.

So, we need to do this conversion because the `date_from` assumes it's in the project timezone?